### DISCUSSION TITLE
FAILLE SSRF SUR REVOLT

### DISCUSSION THREAD

**Discussion by 0xYazuko** [stoatchat/discussions#419] [General]
**Maintainer edit**: this has been marked as invalid, no vulnerability has been found or disclosed.

---

Bonjour, sur votre site moi et mon collègue [0x63, 0x80, 0x6b, 0x73, 0x78]#0001 avons trouvé une faille SSRF sur votre site revolt. Je vous conseille vivement de régler ça parce que j'ai accès à tous le site en entier.
Pour me contacter plus en vigueur veuillez m'ajouter sur discord:
Yazuko#1337

**Comment by Y-v-a-i-n** [comment]
# CWE-918: Server-Side Request Forgery (SSRF)

request : 

```
GET http://h1jxgkfizqzqylxvm0maxozdn4tyhpkdch07nxbm.burpcollaborator.net/ HTTP/1.1

Host: revolt.chat

Pragma: no-cache

Cache-Control: no-cache, no-transform

Connection: close

```

response : 

```

HTTP/1.1 200 OK

Content-Type: text/html

Content-Length: 63

Connection: close

Server: Burp Collaborator https://burpcollaborator.net/

X-Collaborator-Version: 4



<html><body>87frqc0ssrnr2zxs236uzgzjmgjvgkwgifigz</body></html>

```

**Reply by insertish** [reply]
Hi, I don't particularly understand what you're trying to prove here.

Also as-per our policy please use security@revolt.chat if you are reporting vulnerabilities.

**Reply by insertish** [reply]
Ok I didn't see the parent thread nor can I edit my previous reply so I'll go translate that now.

(The GitHub mobile app is quite poor with discussions)

**Comment by insertish** [comment]
(I've moved up to the parent thread)

Hi, if you have something to report please contact us at security@revolt.chat.

**Reply by Y-v-a-i-n** [reply]
I just sent the SSRF vulnerability information to security@revolt.chat x))
